src/amp/pages/Article.tsx

```diff
@@ -9,6 +9,8 @@ import { AdConsent } from '@root/src/amp/components/AdConsent';
 import { css } from 'emotion';
 import { Sidebar } from '@root/src/amp/components/Sidebar';
 import { Analytics, AnalyticsModel } from '@root/src/amp/components/Analytics';
+import { Experiment, } from '@root/src/amp/components/Experiment';
```

Comment:
extra `,`, check if prettier is working correctly on your IDE, this should be auto removed

Reply:
This is after using Intellij. I'll move back to VS code and make sure to run the Prettier formatter on it. Thanks for catching this.